readme.txt

```diff
@@ -1,10 +1,15 @@
 log4stash
 =====================
 
-Breaking chagens on 2.0.4 version:
+Breaking changes on 3.0.0 version:
+BasicAuthUsername and BasicAuthPassword no longer exist as parameters and now needs to be configured through an authentication method.
+DocumentIdSource no longer exists as a parameter and now needs to be configured through IndexOperationParams
+Stopped support of .net 4.0
+
+Breaking changes on 2.0.4 version:
 BasicAuthUsername and BasicAuthPassword moved under AuthenticationMethod, see config example for more information.
 
-Breaking chagens on 2.0.0 version:
+Breaking changes on 2.0.0 version:
```

Comment:
I think we can drop this section. 
This is the readme that will be shown after nuget install/update
I someone still uses <2.0.0 version he can find the details under `breaking.md`